server/src/test/java/org/elasticsearch/discovery/AbstractDisruptionTestCase.java

```diff
@@ -56,27 +52,30 @@
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
-import static org.elasticsearch.discovery.DiscoveryModule.DISCOVERY_HOSTS_PROVIDER_SETTING;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.not;
 
 public abstract class AbstractDisruptionTestCase extends ESIntegTestCase {
 
     static final TimeValue DISRUPTION_HEALING_OVERHEAD = TimeValue.timeValueSeconds(40); // we use 30s as timeout in many places.
 
-    private NodeConfigurationSource discoveryConfig;
+    static final Settings DEFAULT_SETTINGS = Settings.builder()
```

Comment:
Nit: could this move back down to where it was before, since it's not being changed? Would make the diff smaller.

Reply:
done :)